Add cloud related release notes (#1960)
ballerina-platform/ballerina-release: release-notes/swan-lake-2201.1.0-release-note.md

### Code to Cloud Updates

#### Improvements
- Reduced the package size
- Docker image generation now relies on the user's docker client

#### Bug Fixes

To view bug fixes, see the GitHub milestone for Swan Lake 2201.1.0 of the repositories below.

- [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+2201.1.0%22)

> ### `ballerinax/awslambda` Package

Reviewer: ```suggestion
### `ballerinax/awslambda` package
```

#### Improvements

> - Removed from the ballerina distribution. For existing projects with Dependencies.toml change the version into 2.1.0.

Reviewer: ```suggestion
- Removed from the Ballerina distribution. For existing projects, change the version to 2.1.0 in the `Dependencies.toml` file.
```
Removed what? Please specify.

> ### `ballerinax/azure_functions` Package

Reviewer: ```suggestion
### `ballerinax/azure_functions` package
```

#### Improvements

> - Removed from the ballerina distribution. For existing projects with Dependencies.toml change the version into 2.1.0.

Reviewer: ```suggestion
- Removed from the Ballerina distribution. For existing projects, change the version to 2.1.0 in the `Dependencies.toml` file .
```
Removed what? Please specify.
